ietf/secr/templates/includes/sessions_request_form.html

```diff
@@ -23,33 +23,18 @@
              <table>
                <tr>
                  <td colspan="2">Other WGs that included {{ group.name }} in their conflict lists:</td>
-                 <td>{{ session_conflicts }}</td>
-               </tr>
-               <tr class="bg1">
-                 <td rowspan="3" valign="top" width="220">WG Sessions:<br>You may select multiple WGs within each priority</td>
-                 <td width="320">Chair Conflict:</td>
-                 <td>{{ form.wg_selector1 }}
-                     {% comment %}<input type="button" value="Delete the last entry" onClick="delete_last1(); return handleconflictfield(1);"><br>{% endcomment %}
-                     <input type="button" value="Delete the last entry" onClick="delete_last1(); return 1;"><br>
-                     {{ form.conflict1.errors }}{{ form.conflict1 }}
-                 </td>
-               </tr>
-               <tr class="bg1">
-                 <td>Technology Overlap:</td>
-                 <td>{{ form.wg_selector2 }}
-                     {% comment %}<input type="button" value="Delete the last entry" onClick="delete_last2(); return handleconflictfield(2);"><br>{% endcomment %}
-                     <input type="button" value="Delete the last entry" onClick="delete_last2(); return 1;"><br>
-                     {{ form.conflict2.errors }}{{ form.conflict2 }}
-                 </td>
-               </tr>
-               <tr class="bg1">
-                 <td>Key Participant Conflict:</td>
-                 <td>{{ form.wg_selector3 }}
-                     {% comment %}<input type="button" value="Delete the last entry" onClick="delete_last3(); return handleconflictfield(3);"><br>{% endcomment %}
-                     <input type="button" value="Delete the last entry" onClick="delete_last3(); return 1;"><br>
-                     {{ form.conflict3.errors }}{{ form.conflict3 }}
-                 </td>
+                 <td>{{ session_conflicts.inbound }}</td>
                </tr>
+               {% for cname, cfield, cselector  in form.wg_constraint_fields %}
+                 <tr class="bg1">
+                   {% if forloop.first %}<td rowspan="3" valign="top" width="220">WG Sessions:<br>You may select multiple WGs within each priority</td>{% endif %}
```

Comment:
Is `priority` the right word here?  Wouldn't it be something more like `for each category`?

Reply:
Category sounds better